server.js

```diff
@@ -217,6 +217,16 @@ app.post('/api/v1/races', async (request, response) => {
   }
 });
 
+app.delete('/api/v1/characters/:id', async (request, response) => {
+  try {
+    await database('characters').where('id', request.params.id).del();
+    const characters = await database('characters').select();
+    response.status(202).json({ characters });
```

Comment:
if it's successful all remained character array is returned in response with status code `200`